pkg/driverbuilder/builder/builders.go

```diff
@@ -305,13 +368,12 @@ func Targets() []string {
 func (c Config) toTemplateData(b Builder, kr kernelrelease.KernelRelease) commonTemplateData {
 	c.setGCCVersion(b, kr)
 	return commonTemplateData{
-		DriverBuildDir:    DriverDirectory,
-		ModuleDownloadURL: fmt.Sprintf("%s/%s.tar.gz", c.DownloadBaseURL, c.DriverVersion),
```

Comment:
`ModuleDownloadURL` is only needed at libs download time, therefore it is now part of `libsDownloadTemplateData`.